Changes

```diff
@@ -1,5 +1,16 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+
+4.1.1 2025-06-26 (part of Zonemaster v2025.1 release)
+
+ [Fixes]
+- Updates Dockerfile for release 2025.1 #229
+- Sets minimum Perl version to v5.26.0 #228
```

Comment:
Isn't bumping the Perl requirement a major change?

Reply:
> Isn't bumping the Perl requirement a major change?

It is tagged with V-patch, and as I recall we agreed on that on a meeting since even 5.26 is old. But I could agree with you. It is, by definition, a breaking change. I'll change.

@tgreenx, @marc-vanderwal, @tolvmannen, @MichaelTimbert, please comment!